cmd/sippy-daemon/main.go

```diff
@@ -25,10 +25,11 @@ import (
 var logLevel = "info"
 
 type SippyDaemonFlags struct {
-	BigQueryFlags    *flags.BigQueryFlags
-	CacheFlags       *flags.CacheFlags
-	DBFlags          *flags.PostgresFlags
-	GoogleCloudFlags *flags.GoogleCloudFlags
+	BigQueryFlags       *flags.BigQueryFlags
+	CacheFlags          *flags.CacheFlags
+	DBFlags             *flags.PostgresFlags
+	GoogleCloudFlags    *flags.GoogleCloudFlags
+	GoogleStorageBucket string
```

Comment:
Sippy daemon keeps this flag, because it walks the buckets.  It's gone from everywhere else.